package.json

```diff
@@ -42,7 +42,8 @@
     ],
     "copyright": "©2017 Kandra Labs, Inc.",
     "mac": {
-      "category": "public.app-category.productivity"
+      "category": "public.app-category.productivity",
+      "asarUnpack": "./app/node_modules/node-notifier/vendor/**"
```

Comment:
@cPhost do we need to unpack this for `node-mac-notifier` also?

Reply:
No we do need to.